SerialPrograms/Source/PokemonSV/Options/PokemonSV_SandwichMakerOption.cpp

```diff
@@ -0,0 +1,136 @@
+            {SandwichRecipe::shiny_flying,      "shiny_flying",     "Sparkling + Title + Encounter: Flying"},
+            {SandwichRecipe::shiny_psychic,     "shiny_psychic",    "Sparkling + Title + Encounter: Psychic"},
+            {SandwichRecipe::shiny_bug,         "shiny_bug",        "Sparkling + Title + Encounter: Bug"},
+            {SandwichRecipe::shiny_rock,        "shiny_rock",       "Sparkling + Title + Encounter: Rock"},
+            {SandwichRecipe::shiny_ghost,       "shiny_ghost",      "Sparkling + Title + Encounter: Ghost"},
+            {SandwichRecipe::shiny_dragon,      "shiny_dragon",     "Sparkling + Title + Encounter: Dragon"},
+            {SandwichRecipe::shiny_dark,        "shiny_dark",       "Sparkling + Title + Encounter: Dark"},
+            {SandwichRecipe::shiny_steel,       "shiny_steel",      "Sparkling + Title + Encounter: Steel"},
+            {SandwichRecipe::shiny_fairy,       "shiny_fairy",      "Sparkling + Title + Encounter: Fairy"},
+            {SandwichRecipe::huge_normal,       "huge_normal",      "Sparkling + Title + Humungo: Normal"},
+            {SandwichRecipe::custom,            "custom",           "Custom Sandwich"},
+        },
+        LockWhileRunning::LOCKED,
+        SandwichRecipe::shiny_normal
+        )
+    , HERBA_ONE(
+        "<b>Herba Mystica:</b><br>Select the Herba Mystica to use in the preset recipe. Keep in mind invalid herb combinations.",
+        {
+            {HerbaSelection::sweet_herba_mystica,   "sweet-herba-mystica",  "Sweet Herba Mystica"},
+            {HerbaSelection::salty_herba_mystica,   "salty-herba-mystica",  "Salty Herba Mystica"},
+            {HerbaSelection::sour_herba_mystica,    "sour-herba-mystica",   "Sour Herba Mystica"},
+            {HerbaSelection::bitter_herba_mystica,  "bitter-herba-mystica", "Bitter Herba Mystica"},
+            {HerbaSelection::spicy_herba_mystica,   "spicy-herba-mystica",  "Spicy Herba Mystica"},
+        },
+        LockWhileRunning::LOCKED,
+        HerbaSelection::salty_herba_mystica
+    )
+    , HERBA_TWO(
+        "",
+        {
+            {HerbaSelection::sweet_herba_mystica,   "sweet-herba-mystica",  "Sweet Herba Mystica"},
+            {HerbaSelection::salty_herba_mystica,   "salty-herba-mystica",  "Salty Herba Mystica"},
+            {HerbaSelection::sour_herba_mystica,    "sour-herba-mystica",   "Sour Herba Mystica"},
+            {HerbaSelection::bitter_herba_mystica,  "bitter-herba-mystica", "Bitter Herba Mystica"},
+            {HerbaSelection::spicy_herba_mystica,   "spicy-herba-mystica",  "Spicy Herba Mystica"},
+        },
+        LockWhileRunning::LOCKED,
+        HerbaSelection::salty_herba_mystica
+        )
+    , SANDWICH_INGREDIENTS("<b>Custom Sandwich:</b><br>Make a sandwich from the selected ingredients.<br>You must have at least one ingredient and one condiment, and no more than six ingredients and four condiments.")
```

Comment:
could you add a validation for this ?

Reply:
there's validation for ingredient selection in PokemonSV_SandwichMaker.cpp